acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/comparison/CompareSignMessageAcceptanceTest.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.eth2signer.tests.comparison;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static tech.pegasys.eth2signer.dsl.lotus.FilecoinJsonRequests.walletSignMessage;
+import static tech.pegasys.eth2signer.dsl.lotus.LotusNode.OBJECT_MAPPER;
+
+import tech.pegasys.eth2signer.core.service.jsonrpc.FilecoinSignedMessage;
+
+import java.math.BigInteger;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.units.bigints.UInt64;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class CompareSignMessageAcceptanceTest extends CompareApisAcceptanceTestBase {
+
+  private static final Logger LOG = LogManager.getLogger();
+
+  @BeforeEach
+  void initSigner() {
+    super.initAndStartSigner(true);
+  }
+
+  @Test
+  void signMessage() {
+
+    for (int i = 0; i < 50; i++) {
+      LOG.info("################### PERFORMING RUN {}", i);
+      addressMap
+          .entrySet()
```

Comment:
addressMap.keySet().parallelStream will give you address.

Reply:
ooh nicer.